include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -485,10 +489,10 @@ struct OrtApi {
 
   /** \brief Create an OrtEnv
   *
-  * \param[in] logging_function
-  * \param[in] logger_param
-  * \param[in] logging_level
-  * \param[in] logid
+  * \param[in] logging_function A user-provided logging function.
+  * \param[in] logger_param Arbitrary user-provided parameter to pass to `logging_function`.
```

Comment:
Maybe 'A pointer passed as the `param` parameter to the `logging_function` function.'?